x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/details_page/index.tsx

```diff
@@ -3,58 +3,212 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import React, { Fragment, useState } from 'react';
-import { Redirect, useRouteMatch } from 'react-router-dom';
+import React, { Fragment, useCallback, useMemo, useState } from 'react';
+import { Redirect, useRouteMatch, generatePath } from 'react-router-dom';
 import { i18n } from '@kbn/i18n';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, FormattedDate } from '@kbn/i18n/react';
 import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiCallOut,
   EuiText,
   EuiSpacer,
   EuiTitle,
-  EuiHealth,
   EuiButton,
   EuiButtonEmpty,
   EuiEmptyPrompt,
-  EuiBadge,
+  EuiI18nNumber,
+  EuiStat,
 } from '@elastic/eui';
+import { Props as EuiTabProps } from '@elastic/eui/src/components/tabs/tab';
 import { useGetOneAgentConfig, useLink } from '../../../hooks';
-import { AGENT_CONFIG_DETAILS_PATH } from '../../../constants';
+import { AGENT_CONFIG_DETAILS_PATH, AGENT_CONFIG_PATH } from '../../../constants';
 import { Datasource } from '../../../types';
 import { Loading } from '../../../components';
-import { ConnectedLink } from '../../fleet/components';
 import { WithHeaderLayout } from '../../../layouts';
-import { AgentConfigDeleteProvider } from '../components';
 import { ConfigRefreshContext, useGetAgentStatus, AgentStatusRefreshContext } from './hooks';
-import { DatasourcesTable, DonutChart, EditConfigFlyout } from './components';
+import { DatasourcesTable, EditConfigFlyout } from './components';
+import { LinkedAgentCount } from '../components';
+
+const DETAILS_ROUTER_PATH = ':configId/:tabId';
 
 export const AgentConfigDetailsPage: React.FunctionComponent = () => {
   const {
-    params: { configId },
-  } = useRouteMatch<{ configId: string }>();
+    params: { configId, tabId = '' },
+    // path: currentRoutePath,
+  } = useRouteMatch<{ configId: string; tabId?: string }>();
   const agentConfigRequest = useGetOneAgentConfig(configId);
   const agentConfig = agentConfigRequest.data ? agentConfigRequest.data.item : null;
   const { isLoading, error, sendRequest: refreshAgentConfig } = agentConfigRequest;
-  const [redirectToAgentConfigList, setRedirectToAgentConfigsList] = useState<boolean>(false);
+  const [redirectToAgentConfigList /* , setRedirectToAgentConfigsList*/] = useState<boolean>(false);
   const agentStatusRequest = useGetAgentStatus(configId);
   const {
-    isLoading: agentStatusIsLoading,
-    error: agentStatusError,
+    // isLoading: agentStatusIsLoading,
```

Comment:
nitpick but we can removes these two lines

Reply:
Yeah - you right. Let me clean up the few commented out code I left behind